action.yaml

```diff
@@ -0,0 +1,42 @@
+name: "Add labels to PR"
+description: "Add labels to PR Action"
+
+inputs:
+  github_owner:
+    description: "Owner of the repository"
+    default: ${{ github.repository_owner }}
+  github_repo:
+    description: "Repository name"
+    default: ${{ github.event.repository.name }}
+  github_token:
+    description: "GitHub token"
+    default: ${{ github.token }}
+  github_pr_number:
+    description: "GitHub PR number"
+    default: ${{ github.event.pull_request.number }}
+  github_pr_title:
+    description: "GitHub PR title"
+    default: ${{ github.event.pull_request.title }}
+
+outputs:
+  added_labels:
+    description: "Added labels to PR"
+    value: ${{ steps.run.outputs.added_labels }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Run script
+      id: run
+      working-directory: .github/.scripts
+      shell: bash
+      run: echo "added_labels=$(./add_labels.sh)" >> $GITHUB_OUTPUT
```

Comment:
we can move output echo right to the script

Reply:
also you can extract pr title in another step (using existing actions) and pass it as input